### DISCUSSION TITLE
Choose colors palette to convert image to SVG

### DISCUSSION THREAD

**Discussion by merdekiti** [visioncortex/vtracer#88] [Ideas]
Hello. I have an Idea. Is it possible to choose a palette of colors (for example 200 colors RGB or HEX) to use when converting an image into an SVG file?

**Comment by tyt2y3** [comment]
it certainly is. it doesn't not necessarily have to be implemented inside vtracer though, it can be a pre-processing pass on the image

**Reply by merdekiti** [reply]
Sure. But the problem is that vtracer doesn't strictly follow colors of the original image. Means, it converts image into SVG with similar colors but not exact colors as on the original image

**Reply by merdekiti** [reply]
If there is an option to strictly use colors from the original image, it would be helpful

**Reply by tyt2y3** [reply]
well then, I get it. it's possible to map the clusters' colors to a closest one given a palette.
but using the original colors from the image wont work, because the image is likely to have noisy colors due to anti-aliasing or compression.

**Reply by merdekiti** [reply]
I see. In this case, if I can provide a list of colors that should be used during the conversion, it's also perfect solution

**Reply by merdekiti** [reply]
Could you tell me if it's possible to add this feature to your backlog?

**Reply by tyt2y3** [reply]
backlog, sure why not. I think it's not the first time being asked IIRC. if you are willing to sponsor the development, it will surely be implemented soon!

**Reply by merdekiti** [reply]
Can we discuss how much it can cost?

**Reply by tyt2y3** [reply]
can you email me, on my profile, chris.2y3 (at) outlook
